src/Stratis.SmartContracts/CreateContract.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Stratis.SmartContracts
+{
+    public sealed class CreateContract
```

Comment:
Think this *and* TransferFundsToContract should be named better.

Reply:
Can we remove this class and pass the params directly into `Create`? Removes an object allocation, simplifies whitelisting, keeps our API surface smaller, prevents confusion about what types can and cannot be instantiated, etc.